script/Ciel/Day 7/07_01_CIEL07_1_3.txt

```diff
@@ -113,43 +116,44 @@ She's mad at me for ignoring the fact that she specifically told me to come talk
 [offset:15047]{
 -- Page 98, Offset 15047.
 -- それはともかく。
-But I can put that aside for now.
+But even if she is―――
 }
 [sha:698a9e9b53a113c192eb0b0b02f5a598d834165f]{
 -- Page 98, Offset 15048.
 -- 「カレーパン、あるよ」
-"I've got some curry bread."
+"I've got some curry buns."
 }
 [sha:9b1060157f0cd9bc81055b7449ce2400bf6e2aeb]{
 -- Page 99, Offset 15049.
 -- 「遠野くん、わたしが食べ物につられる人間だって思ってるんですかっ！？」
-"Do you really think I'm the kind of person who'd be swayed by some food?!"
+"Tohno-kun, do you really think I'm the kind of person who'd be swayed by some food?!"
 }
 [sha:93ca948d6367dff7b389df68f7b30d222a3e7130]{
 -- Page 99, Offset 15050.
 -- 扉ごしに先輩が怒鳴る。
-Senpai loudly exclaims through the door.
+Senpai yells at me through the door.
 }
 [sha:c20d12bcfff3813a1e0c047d42012c654de061fe]{
 -- Page 99, Offset 15051.
 -- 「……む」
-"Hm."
+"%{i}Geh%{/i}..."
 }
 [sha:2fb75782768fcc3c35c41d045af9222e6975e86b]{
 -- Page 99, Offset 15052.
 -- しまった。反応はいまいちどころか逆効果だ。
-Drats. Not only did it not work, it's made things worse.
+Crap. My plan backfired.
 }
 [sha:5cc1ae23fe145d9664f935de1b275cfdec2706cf]{
 -- Page 100, Offset 15053.
 -- ……まいったな。このまま強引に入ったらますます怒られそうだし、やはり放課後まで先輩と話はできないのか……。
-What do I do now? If I force myself in, she'll probably get even madder at me.
- Suppose I have no choice but to wait until classes are over after all...
+What do I do now?
+ If I force my way in, she'll only get even angrier at me.
+ I guess I'll have to wait until after school after all...
 }
 [sha:ad381193b02abe9168ac964b5529534bbce09a0a]{
 -- Page 101, Offset 15054.
 -- 「まあ、それはそれでもらっておきますが」
-"Though, well, that said, I suppose I could take you up on the offer."
+"Though, since you're here, I suppose I could take you up on the offer."
```

Comment:
Not really a fan of 'it is what it is'. Feels like a bit of a weird connection to her prev sentence. I read the intent here as 'well, despite white I've just said', 'be that as it may', etc. and chose 'since you're here' in that vein.

Reply:
I like the current TL. I think it's a natural parallel to the jp